modules/installation-extend-edge-nodes-aws-local-zones.adoc

```diff
@@ -0,0 +1,93 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_aws/installing-aws-localzone.adoc
+
+ifeval::["{context}" == "installing-aws-localzone"]
+:localzone:
+endif::[]
+
+:_content-type: PROCEDURE
+[id="installation-extend-edge-nodes-aws-local-zones_{context}"]
+= Creating user workloads in AWS Local Zones
+After you create an Amazon Web Service (AWS) Local Zone environment, and you deploy your cluster, you can use edge worker nodes to create a user workloads in Local Zone subnets. 
```

Comment:
@mtulio I've documented the creation of a `deployment` resource and `service` resource. Where do these get created? Inside a cluster that operates in a specific Local Zone subnet? Do the resources apply to all edhe worker nodes that operate inside the cluster?

Reply:
> Where do these get created? Inside a cluster that operates in a specific Local Zone subnet?
The pods will be created on the `namespace` defined on the deployment and the node (Instance running in Local Zone location/subnet) selected by `nodeSelector`

> Do the resources apply to all edhe worker nodes that operate inside the cluster?

No, only for the nodes matching the `nodeSelector` with tainted nodes matching the `tolerations`.

Reply:
Thanks. I added the following callout:

> `nodeSelector`: Targets edge worker nodes that match the specified toleration.